### ISSUE TITLE
Table10 and Zero-shot performance

### ISSUE THREAD

**Issue by hsb1995** [AIoT-MLSys-Lab/SVD-LLM#25]
Our team highly recognizes your work. You are currently the best at matrix decomposition.
How can I reproduce the generation task in Table 10?

I have read your code in detail. Is there no such code in the Zero-shot task?

**Comment by SpeeeedLee** [comment]
Have similar problem, I was wandering about how to prompt the pre-trained LM so that the response is in a good and reasonable format for your accuracy evaluation reported in the paper, thanks.

**Comment by hsb1995** [comment]
> Have similar problem, I was wandering about how to prompt the pre-trained LM so that the response is in a good and reasonable format for your accuracy evaluation reported in the paper, thanks.

Hello, this paper states that no matter what, the. pt file is saved in the end, and then Lora needs to fine tune it. He said that the fine-tuning code is copied from LLM Pruner, and we can see how LLM Pruner evaluates zero shot to solve this problem.
